packages/perspective-viewer-d3fc/src/js/legend/styling/resizableComponent.js

```diff
@@ -98,30 +105,43 @@ export function resizableComponent() {
             const offset = enforceDistToParallelBarConstraints(enforceContainerBoundaries(leftHandle.node(), event.x, 0).x, handles, "width", (x, y) => x - y);
             containerNode.style.left = `${containerNode.offsetLeft + offset}px`;
             containerNode.style.width = `${containerNode.offsetWidth - offset}px`;
+            updateSettings();
```

Comment:
It's a minor shame to have to call this in every drag<Direction>. In the case of dragging a corner it will update the settings twice (oh god the performance implications!). That being said I can't really see a cleaner way to do this.

Reply:
I guess you could have it only update the relevant settings? Althought, as you stay, corner dragging would still be called twice. I'm not sure this would really achieve anything other than making the code a little bit messier.